core/src/main/scala/org/http4s/internal/parsing/Rfc7235.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s.internal.parsing
+
+import cats.parse.Parser.charIn
+import cats.parse.Parser1
+import cats.parse.Rfc5234.{alpha, digit}
+
+private[http4s] object Rfc7235 {
```

Comment:
there are other parsers in RFC7235 that might find these useful, If this is premature I can fold it back to the Authorization parser

Reply:
No, I think this is good now.